examples/generate_ds_transform.py

```diff
@@ -0,0 +1,36 @@
+import random
+import numpy as np
+
+import hub
+from hub.schema import Image, ClassLabel
+
+
+# Create a new dataset
+schema = {
+    "image": Image(shape=(None, None, None), max_shape=(100, 100, 3), dtype="uint16"),
+    "label": ClassLabel(num_classes=3),
+    "id": "uint64",
+}
+tag = "/tmp/ds_classification"
+len_ds = 100
+ds = hub.Dataset(tag, mode="w+", shape=(len_ds,), schema=schema)
+
+
```

Comment:
Instead of this can we use a real-world dataset? I am okay with kaggle datasets or any dataset we already have.